packages/block-library/src/column/edit.native.js

```diff
@@ -133,7 +133,7 @@ function ColumnEdit( {
 			);
 		}
 		return null;
-	}, [ contentStyle[ clientId ], screenWidth, isSelected, hasChildren ] );
+	}, [ contentStyle, clientId, screenWidth, isSelected, hasChildren ] );
```

Comment:
The value passed as `contentStyle` (i.e. `contentWidths`) is memoized here:
https://github.com/WordPress/gutenberg/blob/1ae7d9f72b2b8762753eee88ccc3d9cbfe6070b4/packages/block-library/src/columns/edit.native.js#L147-L157
https://github.com/WordPress/gutenberg/blob/1ae7d9f72b2b8762753eee88ccc3d9cbfe6070b4/packages/block-library/src/columns/edit.native.js#L290

Hence, using this new dependency list won't produce extra renders.

Reply:
Yup! Since it's still a dependency I left it to also avoid the ESLint warning, although it can be disabled it's best to leave it.